src/components/OpenIssue.jsx

```diff
@@ -17,10 +17,10 @@ const OpenIssue = (props) => {
 
   return (
     <div className={classes}>
-      <a href={props.data.html_url}>
+      <a href={props.data.html_url} target="_blank">
         <strong>{props.data.title}</strong> <small>{` #${props.data.number} `}</small>
       </a>
-      { labels }
+      {labels}
```

Comment:
Do you mind add the spaces back in here inside the brackets? I usually tend to include a space in between the bracket and text when it is some actually renderable content (even though I know I didn't do that on line 21. ex:

```
<div>
  { items } 
</div>
```

I usually don't include spaces inside the bracket when I am inlining a javascript value for an html attribute or JSX prop. Ex:

```
<SampleComponent data={someData} />
```